SwiftGRPC.podspec

```diff
@@ -0,0 +1,53 @@
+# Copyright 2018, gRPC Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#   http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Pod::Spec.new do |s|
+    s.name = 'SwiftGRPC'
+    s.module_name = 'GRPC'
+    s.version = '1.0.0-alpha.10'
+    s.license     = { :type => 'Apache License, Version 2.0',
+                      :text => <<-LICENSE
+                        Copyright 2018, gRPC Authors. All rights reserved.
+                        Licensed under the Apache License, Version 2.0 (the "License");
+                        you may not use this file except in compliance with the License.
+                        You may obtain a copy of the License at
+                          http://www.apache.org/licenses/LICENSE-2.0
+                        Unless required by applicable law or agreed to in writing, software
+                        distributed under the License is distributed on an "AS IS" BASIS,
+                        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+                        See the License for the specific language governing permissions and
+                        limitations under the License.
+                      LICENSE
+                    }
+  
+    s.summary = 'Swift gRPC code generator plugin and runtime library'
+    s.homepage = 'https://www.grpc.io'
+    s.authors  = { 'The gRPC contributors' => 'grpc-packages@google.com' }
+    s.source = { :git => 'https://github.com/grpc/grpc-swift.git', :tag => s.version }
+  
+    s.swift_versions = ['4.2', '5']
+    s.requires_arc = true
+    s.ios.deployment_target = '10.0'
+    s.osx.deployment_target = '10.12'
+    s.tvos.deployment_target = '10.0'
+  
+    s.source_files = 'Sources/GRPC/*.swift', 'Sources/GRPC/**/*.swift'
+  
+    s.dependency 'CGRPCZlib', s.version.to_s
+    s.dependency 'SwiftNIO', '~> 2.0'
+    s.dependency 'SwiftNIOHTTP2', '~> 1.0'
+    s.dependency 'SwiftNIOTLS', '~> 2.0'
+    s.dependency 'SwiftNIOSSL', '2.7.0'
+    s.dependency 'SwiftNIOTransportServices', '~> 1.0'
```

Comment:
Can you use the minimum versions from Package.swift?

Reply:
So went down the road for the last few nights of scripting this leveraging a similar pattern from the other SwiftNIO repositories. This would work, up until the point where we need to add `SwiftNIOTLS`. As this is not an explicit dependency in the `Package.swift`, I'm unable to think of a way that I'm happy with without having to hard code this one. 

Any suggestions or preferences?

Reply:
What about taking the current versions from `Package.lock` whenever the Podfile is generated?

Reply:
Didn't think to look in the `Package.resolved`! I've been using `swift package dump-package`.

Was just was looking through it and `SwiftNIOTLS` is the only one that doesn't appear in either the Package or resolved because it is pulled in under `SwiftNIO`. I can hard code the script to add `SwiftNIOTLS` and use the `SwiftNIO` version for just that dependency, but it just won't be as clean and wanted to see if there was a way to handle it more efficiently. 🤔

Reply:
Scratch that, my bad! Turns out it doesn't explicitly require `SwiftNIOTLS` - sticking with the `Package.resolved` for acquiring the versions approach! I'll finalize the script this weekend.

Sorry for the confusion/thanks for the guidance @MrMage 👍